.github/workflows/build.yaml

```diff
@@ -97,6 +97,14 @@ jobs:
         if: ${{ matrix.shortcut == 'cs9' }}
         run: pip3 install . -U
 
+      - name: Checkout target repository
+        uses: actions/checkout@v2
```

Comment:
You need to add a condition here as this can be done only during push patch, but not on PR:

```
if: ${{ github.event_name == 'push' }}
```

Reply:
Why it should not be on the PR? This change only affects the generated docs which we do in the task below `Create python documentation` and on both push and PR

Reply:
So why it fails then?

https://github.com/oVirt/python-ovirt-engine-sdk4/runs/4850792026?check_suite_focus=true

Reply:
It does not see the secrets, maybe some precaution for PR? When I was testing it on my fork the changes were working https://github.com/mnecas/python-ovirt-engine-sdk4/actions/runs/1709883972

Reply:
Ah, so do we need to copy also https://github.com/oVirt/python-ovirt-engine-sdk4/blob/main/.github/workflows/build.yaml#L133 into the build job?

Reply:
Nope that config is for https://github.com/oVirt/python-ovirt-engine-sdk4/blob/main/.github/workflows/build.yaml#L168 I should move it right next to it

Reply:
The issue is by design https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflows-in-forked-repositories
I'll move the template to this repo and update the PR.